components/honoraryMember.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { Stack, Box, Tooltip } from "@mui/material";
+import styles from "../styles/Member.module.css";
+import { isMobile } from "react-device-detect";
+import Image from "next/image";
+
+export interface MemberProps {
+  name: string;
+  key: string;
+  memberInfo: string;
+  profilePic: string;
+  reverse: boolean;
+  appointed: string;
+  position: string;
+  committee: string[];
+}
+
+const HonoraryMember = (member: MemberProps) => {
```

Comment:
The file name should also be capitalized, `HonoraryMember.tsx`, to match the component name